Dockerfile

```diff
@@ -14,6 +14,11 @@ WORKDIR protocol
 RUN npm install
 RUN scripts/buildContracts.sh
 
+# Environment variables to set up email notifications.
```

Comment:
nit: can you add these to the sample `docker run` command at the top?

Reply:
Done.